src/Backend/DataSeriesFactory.js

```diff
@@ -0,0 +1,24 @@
+export default class TraceFactory {
+    static getXValuesByPoints(minValue, maxValue, numberOfPoints) {
+        if(numberOfPoints === 1) return [(maxValue - minValue)/2]
```

Comment:
```suggestion
        if(numberOfPoints === 1) return [(maxValue + minValue)/2.0]
```


mean value is calculated as (a+b)/2 not as (a-b)/2